src/rendering/RenderUtil.cc

```diff
@@ -457,6 +520,37 @@ void RenderUtil::Update()
       visual->SetUserData("temperature", temp.second);
     }
   }
+
+  // create new collision visuals
+  {
+    if (!newCollisionLinks.empty())
+      DeselectAllEntities();
```

Comment:
is this done so that the object does not get highlighted with bright white emissive material?

Reply:
This was done to avoid encountering this error after a new collision was created and clicking elsewhere (deselecting an entity) https://github.com/ignitionrobotics/ign-gazebo/blob/4a1cba6d2e32e493c5dc0f7233db01b406681adf/src/rendering/RenderUtil.cc#L1427-L1428

Reply:
I tried commenting the above two lines out to see what's causing this error msg appear but I couldn't reproduce it by selecting the model and toggling View > Collision. Can you list the steps to reproduce this?

Reply:
1. Start up Ignition with a sample world
2. Select a model in Scene3D so that it is highlighted
3. Then right click the model, View > Collisions
4. Then click anywhere else in the scene and the error should appear

The error only appears once, after the collision visual has been created. If you toggle off the collision then run through the above steps again then the error doesn't appear again.

Reply:
Thanks I was able to reproduce the problem now. I think the issue is that It was trying to update the emissive property of a new geometry (collision visual's geom) that it does not previously know about. The current code works around this problem by deselecting all entities before creating the new collision visual. But there would now a mismatch between what's selected in scene 3d and the entity tree (and other plugins) for example. It's more of a usability issue now. I would recommend seeing if there's another way around this without deselecting all entities, e.g. updating the logic in the `LowlightNode` function or simply removing that error msg.

Reply:
Fixed the issue by adding the collision visual's geometry emissive property to the `originalEmissive` map after creation of the collision visual: df7f00942033bc20b6416f76df048df648daca2f